src/VisualStudio/Core/Def/Implementation/ProjectSystem/AbstractProject.cs

```diff
@@ -171,6 +174,12 @@ internal abstract partial class AbstractProject : ForegroundThreadAffinitizedObj
             }
 
             UpdateAssemblyName();
+
+            Logger.Log(FunctionId.AbstractProject_Created,
+                KeyValueLogMessage.Create(LogType.Trace, m =>
+                {
+                    m[ProjectGuidPropertyName] = Guid;
```

Comment:
tagging @jonsequitur since I believe Project Guid is now considered PII

Reply:
Yeah, that was one of the things I wondered about.  Also, in multi-targeting cases we'll have more than one of these per-project, but I'm not sure what to log to distinguish them.  We don't have a concept of TFM here.  Could be `DisplayName` or `OutputPath` I guess - though I'm not sure OutputPath is always set.

Reply:
Project GUIDs are classified as EUPI.

Reply:
Which means what exactly?